vespalib/src/vespa/vespalib/util/count_down_latch.h

```diff
@@ -82,7 +82,10 @@ class CountDownLatch
      *
      * @return current count
      **/
-    uint32_t getCount() const { return _count; }
+    [[nodiscard]] uint32_t getCount() const noexcept {
+        std::unique_lock guard(_lock);
```

Comment:
Enough to use a lock_guard ?

Reply:
Yes, force of habit from spending too much time around condition variables 🙂  Will simplify